docs/source/examples.rst

```diff
@@ -49,6 +49,22 @@ Deconvolution
    examples/deconv_ppp_bm4d_admm
 
 
+Machine Learning
+^^^^^^^^^^^^^^^^
+
+.. toctree::
+   :maxdepth: 1
+
+   examples/ct_astra_datagen_foam2
```

Comment:
Why the `2` at the end of the filename?

Reply:
To differentiate it from a foam that only has two intensity levels (foam and background). A foam2 has two different attenuation properties, and therefore, three intensity levels (foam A, foam B and background).

Reply:
In that case, perhaps call the other one `foam1` so that we have `foam1` and `foam2` rather than `foam` and `foam2`?